### PR TITLE
#11426 dashboard

### PR DESCRIPTION
**Pull request by mahmut-gundogdu**
Ordering dashboard implemented

 closes abpframework/abp#11426

### CODE REVIEW

File: services/ordering/src/EShopOnAbp.OrderingService.EntityFrameworkCore/EShopOnAbp.OrderingService.EntityFrameworkCore.csproj [hunk 1]
```diff
@@ -9,6 +9,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\EShopOnAbp.OrderingService.Domain\EShopOnAbp.OrderingService.Domain.csproj" />
+    <ProjectReference Include="..\EShopOnAbp.OrderingService.Application.Contracts\EShopOnAbp.OrderingService.Application.Contracts.csproj" />
```
Comment (gterdem) [inline services/ordering/src/EShopOnAbp.OrderingService.EntityFrameworkCore/EShopOnAbp.OrderingService.EntityFrameworkCore.csproj:12]: This is not correct. See my comment for `OrderingServiceRemoteServiceConsts`.

File: services/ordering/src/EShopOnAbp.OrderingService.Application.Contracts/OrderingServiceRemoteServiceConsts.cs [hunk 2]
```diff
@@ -1,7 +1,8 @@
 ﻿namespace EShopOnAbp.OrderingService
 {
-    public class OrderingServiceRemoteServiceConsts
+    public static class OrderingServiceRemoteServiceConsts
     {
         public const string RemoteServiceName = "Ordering";
+        public const int Top10 = 10;
```
Comment (gterdem) [inline services/ordering/src/EShopOnAbp.OrderingService.Application.Contracts/OrderingServiceRemoteServiceConsts.cs:6]: This is the wrong layer. Constants should be under Domain.Shared especially when you are planning to use them on EntityFrameworkCore layer.

**Review by gterdem — CHANGES_REQUESTED** [review]

**Review by gterdem — APPROVED** [review]

File: apps/angular/projects/ordering/src/pages/dashboard/components/order-status-chart/order-status-chart.component.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  selector: 'app-order-status-chart',
+  templateUrl: './order-status-chart.component.html',
+})
+export class OrderStatusChartComponent {
+  @Input()
+  set data(value: OrderStatusDto[]) {
+    this.chartData.labels = [...value.map(x => x.orderStatus)];
+    this.chartData.datasets = [
+      {
+        label: 'Order Statuses',
```
Comment (muhammedaltug) [inline apps/angular/projects/ordering/src/pages/dashboard/components/order-status-chart/order-status-chart.component.ts:14]: label needs translation

File: apps/angular/projects/ordering/src/pages/dashboard/components/payments-chart/payments-chart.component.ts [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+@Component({
+  selector: 'app-payments-chart',
+  templateUrl: './payments-chart.component.html',
+})
+export class PaymentsChartComponent {
+  @Input()
+  set data(value: PaymentDto[]) {
+    this.chartData.datasets = [
+      {
+        label: 'Order Statuses',
```
Comment (muhammedaltug) [inline apps/angular/projects/ordering/src/pages/dashboard/components/payments-chart/payments-chart.component.ts:13]: label needs translation

File: apps/angular/projects/ordering/src/pages/dashboard/components/payments-chart/payments-chart.component.ts [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+  chartData = {
+    labels: [],
+    datasets: [],
+  };
+
+  options = {
+    plugins: {
+      title: {
+        display: true,
+        text: 'Payment Methods',
```
Comment (muhammedaltug) [inline apps/angular/projects/ordering/src/pages/dashboard/components/payments-chart/payments-chart.component.ts:30]: text need translation

**Pull request merged**